### ISSUE TITLE
on the final score screen show total Brills earned on left column

### ISSUE THREAD

**Issue by scholar6admin** [Scholar-6/brillder#4471]

**Comment by ivanromakh** [comment]
I am not sure how to do it. needs discussion.

**Comment by ivanromakh** [comment]
@scholar6admin I am not sure there is already a lot of stuff which we added multiple times It will be hard to do,

**Comment by ivanromakh** [comment]
@scholar6admin right not it is showing zero for brills in provisional score as I was trying to fix competition review.

**Comment by ivanromakh** [comment]
@scholar6admin this is problematic not sure what action should be done when user has cashed attempt and trying to save it during review. what brills will he get?

**Comment by ivanromakh** [comment]
@scholar6admin if I try to do it it will make saving attempt even slower that is it now as I will need to get all attempts and get best score among them not sure how long it will take.

**Comment by ivanromakh** [comment]
this will need a lot of testing I am not sure if it will work in every case.

**Comment by ivanromakh** [comment]
I changed best score here to total brills.
![image](https://user-images.githubusercontent.com/18567511/165921291-fab1ad77-b623-44f5-99c3-bb2b1f333740.png)

**Comment by s-steddy** [comment]
Have done my best to understand this and looks like it is mostly done. I take it the total brills bar serves the same purpose as the tooltip that was mentioned?

The only thing missing from the original list is the request for a message saying "including x Brills bonus", if we still want that?
<img width="400" alt="Screenshot 2022-08-17 at 16 06 00" src="https://user-images.githubusercontent.com/107257153/185177588-b6402a00-d111-4dfb-bd7a-db21bda85ee3.png"> 

But I also found some bugs.
i. On second play through, the provisional score creen told me I'd got worse, when I hadn't - I got 51 on first attempt here, and 83 on the new attempt:
<img width="400" alt="Screenshot 2022-08-17 at 16 05 04" src="https://user-images.githubusercontent.com/107257153/185177827-9dbcfcb8-4f09-46a8-a112-a54d79113a33.png">

ii. And clicking around in the review (I was looking for the tooltip), my correct answers were marked in red (see score 8/8 at top):
<img width="400" alt="Screenshot 2022-08-17 at 16 13 09" src="https://user-images.githubusercontent.com/107257153/185178185-da0abe1f-b82c-41bb-8f1b-1dea075308f6.png">

**Comment by ivanromakh** [comment]
@s-steddy as I see in code. we don`t show "including x Brills bonus" anymore.

**Comment by ivanromakh** [comment]
I tried. but can`t reproduce. my first score was 50 and than 75, and it works good.
![image](https://user-images.githubusercontent.com/18567511/185370028-d23daf64-8d65-4975-a32d-e2b94cabcc85.png)

**Comment by ivanromakh** [comment]
maybe backend was busy and update information.

**Comment by ivanromakh** [comment]
ops if you get 0 brills.

**Comment by ivanromakh** [comment]
@s-steddy library users don`t get anybrills. and so they will have that message probably.

**Comment by s-steddy** [comment]
Hmm, but I wasn't playing as a library user! 

But also, library users should still get brills! Just they can't do anything with them... yet

**Comment by ivanromakh** [comment]
for some reason you didn`t get any brills thats why you got that message.

**Comment by ivanromakh** [comment]
@s-steddy which brick it was I will check it myself.

**Comment by ivanromakh** [comment]
ops I can see title:)

**Comment by s-steddy** [comment]
It's one of mine (I knew the answers ...)!

https://app.brillder.com/play/brick/1164/cover/english-language/syllables-and-poetic-structure

**Comment by s-steddy** [comment]
Yeah looking through the images, I have to have earned some brills on the first attempt, so the q is what caused me to earn 0 at the provisional score on the second playthrough?

**Comment by ivanromakh** [comment]
I fixed that bug `ii` from comment above for categorize question. I will check other question types as well.

**Comment by s-steddy** [comment]
It seems that this is working as expected at last. Going to say it's ready to deploy and if it becomes a problem again we can start a new simpler issue

<img width="425" alt="Screenshot 2022-08-26 at 15 22 15" src="https://user-images.githubusercontent.com/107257153/186931076-a101ee43-e019-4385-80ce-15a6d40bc74c.png">
!
